yb-voyager/cmd/importDataStatusCommand.go

```diff
@@ -304,11 +205,8 @@ func prepareRowWithDatafile(dataFile *datafile.FileEntry, snapshotRowCount int64
 		importedCount, err = state.GetImportedByteCount(dataFile.FilePath, dataFile.TableName)
 	} else {
 		totalCount = dataFile.RowCount
-		if importerRole == FB_DB_IMPORTER_ROLE {
-			importedCount = snapshotRowCount
-		} else {
-			importedCount, err = state.GetImportedRowCount(dataFile.FilePath, dataFile.TableName)
-		}
+		importedCount, err = state.GetImportedRowCount(dataFile.FilePath, dataFile.TableName)
```

Comment:
Just checked the code for `state.GetImportedRowCount`. Looks like we're getting the row count from the local state in exportdir. Ideally, I think we should get it from the metadata on the target. (which should be the source of truth). 

Are we doing the same for the live migration report? (i.e. getting it from the local state in exportdir)? 

Let's discuss this once in standup?

Reply:
I think local state can also be a source of truth as we mark a particular batch as Done only when we successfully update target metadata. 
Yes, we are doing the same in live-migration report as well as it saves one overhead of querying DB.